tools/ci_build/github/apple/package_assembly_utils.py

```diff
@@ -15,7 +16,7 @@
 
 
 def gen_file_from_template(template_file: pathlib.Path, output_file: pathlib.Path,
-                           variable_substitutions: dict[str, str]):
+                           variable_substitutions: Dict[str, str]):
```

Comment:
>Dict [](http://example.com/codeflow?start=51&length=4)

I had used typing.Dict before as well, but it seems to be deprecated as of 3.9.
[https://docs.python.org/3/library/typing.html#typing.Dict](https://docs.python.org/3/library/typing.html#typing.Dict)
not sure what Python version we are going with for type annotations. and if you have a way to actually check them please let me know :)

Reply:
I have trouble update my dev env from 3.7 to 3.9, used this to unblock me, and this is more user friendly for python 3.8- users.

Reply:
out of curiosity, what was the blocker?

Reply:
The python on my Mac was from Anaconda, which takes forever to install new version or create new virtual environment